connection_params.ts

```diff
@@ -50,6 +50,7 @@ export interface IConnectionParams {
   user?: string;
   password?: string;
   application_name?: string;
+  cert_file?: string;
```

Comment:
I'm not sure why I used `snake_case` here :| we should probably use `camelCase` for consistency in code style